clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -20,27 +20,22 @@ import { OVERRIDE, stubConfig } from "../support/stubs";
 const PURPOSE_2 = {
   id: 2,
   name: "Use limited data to select advertising",
-  served_notice_history_id: "ser_4120a09f-f6df-4fa4-9563-dd976599f4f3",
```

Comment:
do we need an assertion that we do pass the new top level `served_notice_history_id` in at least one of these tests?